try{
    console.log("there might be no errors");
}catch(error){
    console.log("not catching as there is no error");
}


try{

    console.log("making an error");

    hoyyahoi;

}catch(error){
    console.log("an error encountered");
}